server/athenian/api/precompute/notify_almost_expired_accounts.py

```diff
@@ -36,12 +36,15 @@ async def main(context: PrecomputeContext, args: argparse.Namespace) -> None:
     )
     users = dict(user_rows)
     name_rows = await mdb.fetch_all(
-        select([GitHubAccount.id, GitHubAccount.name]).where(
+        select(GitHubAccount.id, GitHubAccount.name).where(
```

Comment:
I think sqla 1.4 select wants a list just like 1.3 did, are you sure?

Reply:
Yes 1.4 accepts both formats
https://docs.sqlalchemy.org/en/14/changelog/migration_14.html?highlight=select#select-case-now-accept-positional-expressions